javascript/src/main/pages/Admin/SearchInfo.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import { Button } from "react-bootstrap";
+import useSWR from "swr";
+import {useAuth0} from "@auth0/auth0-react";
+import {fetchWithToken} from "../../utils/fetch";
+
+const AppUsers = () => {
```

Comment:
I wonder if the name exported on line 8 should match the name of the file. Either both should be SearchInfo or both should be AppUsers